src/arrayMethodSort.js

```diff
@@ -2,10 +2,42 @@
 
 /**
  * Implement method Sort
+ *
+ *
  */
+
+const defaultCompare = function (a, b) {
+  const aNum = String(a);
+  const bNum = String(b);
+
+  if (aNum < bNum) {
+    return -1;
+  }
+
+  if (aNum > bNum) {
+    return 1;
+  }
+
+  if (aNum === bNum) {
+    return 0;
+  }
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = defaultCompare) {
```

Comment:
The parameter `compareFunction` should be checked to ensure it's a function before it's used. If a non-function argument is passed, the method should throw a TypeError, according to the ECMAScript specification for `Array.prototype.sort`.